source/guide/3.2/guide-policies.md

```diff
@@ -196,6 +263,12 @@ groups:
               # that was previously configured
               workflow: autoheal
```

Comment:
rename autoheal to heal and remove the line stating it was previously configured